src/plugin/sdl/sdl.c

```diff
@@ -786,7 +813,13 @@ static void SDL_handle_events(void)
 	if (config.X_background_pause && !dosemu_user_froze)
 	  freeze_dosemu();
 	break;
+
       case SDL_WINDOWEVENT_RESIZED:
+        if (vga.mode_class == TEXT) {
+          SDL_resize_to_fontsize(event.window.data1, event.window.data2);
```

Comment:
Ah, so you seem to pass the window size, not
the font size... So I was likely confused with the
func name. If it adjusts to the window size (or close
to that), then there is no need to drop anything, just
rename the function to something less confusing.
Like `SDL_resize_to_nearest` or whatever.
The current name suggests the font size is passed.

Reply:
renamed to SDL_resize_to_nearest_fontsize

Reply:
But why `_fontsize`?
Doesn't it take winsize?